.pipelines/js-ci.yml

```
@@ -0,0 +1,104 @@
+name: $(date:yyMM).$(date:dd)$(rev:rrr)
+
+pr:
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - source/nodejs
+
+pool:
+  name: Azure Pipelines
+  vmImage: ubuntu-20.04
+  demands:
+  - npm
+
+parameters:
+- name: target_packages
+  displayName: "Which packages is going to be built."
+  type: object
+  default:
+  - adaptivecards
+  - adaptivecards-controls
+  - adaptivecards-designer
+  - adaptivecards-templating
+  - adaptivecards-react
+
+steps:
+- task: NodeTool@0
+  name: NodeTool1
+  displayName: Use Node 14.x
+  inputs:
+    versionSpec: 14.x
+
+- task: CopyFiles@2
+  inputs:
+    sourceFolder: source/nodejs
+    contents: 'tsconfig.*' 
+    targetFolder: source
+  displayName: Copy tsconfig
+
+- task: CopyFiles@2
+  inputs:
+    sourceFolder: source/nodejs/adaptivecards-react-testapp
+    contents: '**' 
+    targetFolder: source/adaptivecards-react-testapp
+  displayName: Copy adaptivecards-react-testapp
+
+- task: CopyFiles@2
+  inputs:
+    sourceFolder: source/nodejs/adaptivecards-ui-testapp
+    contents: '**' 
+    targetFolder: source/adaptivecards-ui-testapp
+  displayName: Copy adaptivecards-ui-testapp
+
+- bash: |
+   npm ci 
+   npx lerna bootstrap --ci
+   npx lerna run release
+  workingDirectory: source/nodejs
+  displayName: 'Bash - lerna bootstrap'
+
+- bash: |
+   npx lerna run test
+  workingDirectory: source/nodejs
+  displayName: 'Run all tests'
+
+- ${{ each target_package in parameters.target_packages }}:
+  - task: Npm@1
+    displayName: '[${{ target_package }}] npm pack'
+    inputs:
+      command: custom
+      customCommand: pack
+      workingDir: source/nodejs/${{ target_package }}
+  
+  - task: CopyFiles@2
+    inputs:
+      sourceFolder: source/nodejs/${{ target_package }}
+      contents: '*.tgz' 
```

is this going to repeatedly copy files as each `target_package` is built?

should it maybe be:

```suggestion
      contents: '${{target_package}}.tgz' 
```

(here and below)

---

It doesn't work because the package name is something like `ac-typed-schema-0.7.0-alpha.0.tgz`

---

`${{target_package}}*.tgz`?